src/modules/billing/controller.js

```diff
@@ -142,9 +127,25 @@ const deleteAccountFromBatch = async request => {
   };
 };
 
+const deleteBatch = async (request, h) => {
+  const { batch } = request.pre;
+  const batchId = batch.billing_batch_id;
+
+  await batchService.deleteBatch(batchId);
+
+  const evt = await eventService.getEventForBatch(batchId);
```

Comment:
We discussed earlier writing a new event here rather than updating the existing one